config/pal5_config.yaml

```diff
@@ -1,38 +1,53 @@
-# Example configuration file for Pal 5 interpolation
-# from Erkal et al. 2016 []
-stream:
-  filename : &filename ./data/erkal_2016_pal_5_input.csv
-  nstars: 5.0e+3
-   
-  density:
-    type: FileInterpolation
-    filename : *filename
-    columns : [phi1, nstars]
-   
-  track:
-    center:
-      type: FileInterpolation
-      filename: *filename
-      columns: [phi1, phi2]
-    spread:
-      type: FileInterpolation
-      filename: *filename
-      columns: [phi1, width]
-    sampler: Gaussian
-   
 background:
-  nstars : 1.0e+5
-
   density:
     type: Uniform
+    xmax: 18.0
     xmin: -10.0
-    xmax: +18.0
-
+  nstars: 100000.0
   track:
     center:
       type: Constant
       value: 0.0
+    sampler: Uniform
     spread:
       type: Constant
       value: 4.0
+stream:
+  density:
+    columns:
+    - phi1
+    - nstars
+    filename: ./data/erkal_2016_pal_5_input.csv
+    type: FileInterpolation
+  distance_modulus:
+    center:
+      intercept: 16.5
+      slope: 0.1
+      type: Line
     sampler: Uniform
+    spread:
+      type: Constant
+      value: 0.0001
+  filename: ./data/erkal_2016_pal_5_input.csv
+  isochrone:
+    age: 12.0
```

Comment:
I think that we need a line here to specify the survey bandpasses for the isochrone. I think that it would be:
```
    survey: 'lsst'
```
in this case.